shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.cc

```diff
@@ -237,6 +238,23 @@ void FuchsiaExternalViewEmbedder::SubmitFrame(
           }
         }
 
+        // Validate the MutatorsStack encodes the same transform as the
+        // transform matrix.
+        SkMatrix mutatorsTransform;
+        for (auto i = view_params.mutatorsStack().Bottom();
+             i != view_params.mutatorsStack().Top(); ++i) {
+          const auto& mutator = *i;
+          switch (mutator->GetType()) {
+            case flutter::MutatorType::transform: {
+              mutatorsTransform.preConcat(mutator->GetMatrix());
+            } break;
+            default: {
+              break;
+            }
+          }
+        }
```

Comment:
Can we move all this into a DCHECK if that's the only use? maybe move it to a `SkMatrix GetMutatorsTransform(const MutatorStack& mutatorStack)` function in anonymous namespace.